workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -256,19 +256,21 @@ dds.list <- map(lst, lcdbwf::make.dds, salmon.files=NULL, remove.version=TRUE,
 ```{r results, dependson='dds_list', cache=TRUE}
 # Docs: https://lcdb.github.io/lcdb-wf/rnaseq-rmd.html#results
 
-res.list <- list()
-
 # NOTE: Example contrast #1----------------------------------------------------
 #   Using the example data, this compares treatment group to control group.
 #   Change to reflect your experiment.
 # IMPORTANT: Control must always be last
-res.list[['main']] <- list(
+#   Change the letter prefix of the contrast to adjust order of appearance in the html
```

Comment:
This should be added to docs at https://github.com/lcdb/lcdb-wf/blob/master/docs/rnaseq-rmd.rst, and removed here.